src/redux/reducers/data.js

```diff
@@ -34,41 +56,67 @@ export const updateNeighborhoodCouncil = (council) => ({
 });
 
 const initialState = {
-  year: '2015',
-  startMonth: '1',
-  endMonth: '12',
+  startDate: '01/01/2015',
```

Comment:
can these default to null values?

Reply:
startDate and endDate values changed to null.